sse2neon.h

```diff
@@ -2981,6 +2981,16 @@ FORCE_INLINE __m128i _mm_hadd_epi16(__m128i _a, __m128i _b)
 #endif
 }
 
+// Horizontally substract adjacent pairs of single-precision (32-bit)
+// floating-point elements in a and b, and pack the results in dst.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_hsub_ps
+FORCE_INLINE __m128 _mm_hsub_ps(__m128 _a, __m128 _b)
+{
+    float32x4x2_t rev_vec = vuzpq_f32(_a, _b);
```

Comment:
You can provide A64 shortcut as following:
```c
    return vsubq_f32(vuzp1q_f32(_a, _b), vuzp2q_f32(_a,  _b));
```

Check [Neon Intrinsics Reference](https://developer.arm.com/architectures/instruction-sets/simd-isas/neon/intrinsics) for details.